docs/dxp/latest/en/building-applications/objects/building-solutions-with-objects/using-objects-to-create-kanban-boards.md

```diff
@@ -1,3 +1,251 @@
+   | done | done |
+
+   ![Add todo, inprogress, and done to the Task Status Picklist.](./using-objects-to-create-kanban-boards/images/06.png)
+
+After creating the Picklist, you can create an Object that uses this list to set a task's kanban state.
+
+## Creating an Object for the Kanban Tasks
+
+Follow these steps to create an Object for receiving and storing kanban tasks:
+
+1. Open the *Global Menu* (![Global Menu](../../../images/icon-applications-menu.png)), click on the *Control Panel* tab, and go to *Objects*.
+
+1. Click the *Add* button ( ![Add Button](../../../images/icon-actions.png) ) and enter these values.
+
+   | Field | Value |
+   | --- | --- |
+   | Label | Task |
+   | Plural Label | Tasks |
+   | Object Name | Task |
+
+1. Select the *Object*, click on the *Field* tab, and add these four *fields*.
+
+   | Label | Field Name | Type | Picklist | Required |
+   | --- | --- | --- | --- | --- |
+   | Task Title | taskTitle | String | n/a | &#10004; |
+   | Description | description | String | n/a |  |
+   | Task Status | taskStatus | Picklist | Task Status | &#10004; |
+   | Progress | progress | Integer | n/a |  |
+
+   ![Add the Task Title, Description, Task Status, and Progress fields to the Object.](./using-objects-to-create-kanban-boards/images/07.png)
+
+1. Click on the *Details* tab and set the following Entry Display and Scope settings.
+
+   | Field | Value |
+   | --- | --- |
+   | Title Field | Task Title |
+   | Scope | Site |
+   | Panel Category Key | Site Administration > People |
+
+   ![Configure the Object's Entry Display and Scope settings.](./using-objects-to-create-kanban-boards/images/08.png)
+
+1. Click *Publish*.
+
+[Publishing an Object](../creating-and-managing-objects/creating-objects.md#publishing-object-drafts) creates and activates the application with its database columns. This also creates a collection provider for displaying the Object's entries using the Collection Display fragment.
+
+Once Published, you can access the Object and [add task entries](#adding-task-entries) via the Liferay UI or Headless APIs.
+
+## Adding Task Entries
+
+Follow these steps to add task entries to your newly published Object via the Liferay UI:
+
+1. Navigate to the default *Liferay DXP* Site, open the *Site Menu* ( ![Site Menu](../../../images/icon-menu.png) ), and go to *People* &rarr; *Tasks*.
+
+1. Use the *Add* button ( ![Add Button](../../../images/icon-add.png) ) to create task entries.
+
+   Due to limitations in the provided kanban fragment, you'll need at least one task for each Task Status.
+
+   ![Add three tasks to the Object.](./using-objects-to-create-kanban-boards/images/09.png)
+
+After adding task entries, you can [build a Page](#building-a-page-for-displaying-tasks) for displaying them.
+
+## Building a Page for Displaying Tasks
+
+Follow these steps to create a kanban board Page:
+
+1. Navigate to the default *Liferay DXP* Site and create a Content Page named *Tasks*.
```

Comment:
It seemed mandatory in the Add Page dialog to select a template. So I selected _Blank_. Was that the correct choice? I think it might be good to mention a basic template to use, if that is needed. I didn't see any option for Content Page.